<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCPChatClientClass</class>
 <widget class="QMainWindow" name="TCPChatClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCPChatClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>310</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::WidgetWidth</enum>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>578</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>Main</string>
    </property>
    <addaction name="actionClear"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCustom">
    <property name="title">
     <string>Custom</string>
    </property>
    <widget class="QMenu" name="menuColor">
     <property name="title">
      <string>Color</string>
     </property>
     <addaction name="actionRed"/>
     <addaction name="actionGreen"/>
     <addaction name="actionBlue"/>
     <addaction name="actionMagenta"/>
     <addaction name="actionYellow"/>
     <addaction name="actionCyan"/>
     <addaction name="actionBlack"/>
     <addaction name="actionWhite"/>
     <addaction name="actionGray"/>
    </widget>
    <widget class="QMenu" name="menuFont_SIze">
     <property name="title">
      <string>Font SIze</string>
     </property>
     <addaction name="action8"/>
     <addaction name="action10"/>
     <addaction name="action12"/>
     <addaction name="action14"/>
     <addaction name="action16"/>
     <addaction name="action18"/>
    </widget>
    <addaction name="actionName"/>
    <addaction name="menuColor"/>
    <addaction name="menuFont_SIze"/>
    <addaction name="actionShow_Time"/>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuCustom"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionName">
   <property name="text">
    <string>Name</string>
   </property>
  </action>
  <action name="actionRed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </action>
  <action name="actionGreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </action>
  <action name="actionBlue">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </action>
  <action name="actionBlack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Black</string>
   </property>
  </action>
  <action name="actionWhite">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>White</string>
   </property>
  </action>
  <action name="actionCyan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cyan</string>
   </property>
  </action>
  <action name="actionYellow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Yellow</string>
   </property>
  </action>
  <action name="actionMagenta">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Magenta</string>
   </property>
  </action>
  <action name="actionGray">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gray</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionShow_Time">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Time</string>
   </property>
  </action>
  <action name="action8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="action10">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </action>
  <action name="action12">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </action>
  <action name="action14">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>14</string>
   </property>
  </action>
  <action name="action16">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>16</string>
   </property>
  </action>
  <action name="action18">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>18</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TCPChatClient.qrc"/>
 </resources>
 <connections/>
</ui>
